stock_analysis/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# © 2020 Lorenzo Battistini - TAKOBI
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Analysis",
+    "summary": "Analysis view for stock",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Inventory, Logistic, Storage",
```

Comment:
Change this to 'Warehouse' as it won't appear in good category on OCA shop.

Reply:
Thanks, done